packages/ui/core-components/src/lib/unsorted/ui/QueryViewerSupport/prismtheme.css

```diff
@@ -88,7 +88,7 @@ pre[class*='language-'] {
 }
 
 .token.comment {
-	color: var(--blue-700);
+	color: #1d4ed8;
```

Comment:
Was this meant to be hard coded?

Reply:
Yes. I removed the CSS color vars and left this here to maintain our prism theme